src/BookList.js

```diff
@@ -27,13 +28,22 @@ export default function BookList() {
   if (loading) return <div>Loading...</div>;
   if (error) return <p>ERROR</p>;
   return (
-    <List>
-      {data.getBooks.map(({ id, title, author }) => (
-        <ListItem key={id}>
-          <Typography variant='body1' className={classes.title}>{title}</Typography>
-          <Typography variant='subtitle2' color='primary'>{author}</Typography>
-        </ListItem>
-      ))}
-    </List>
+      <List>
+        {data.getBooks.map(({ id, title, author, numPages }) => (
+          <Grid className={classes.root} container alignItems='flex-end' direction='row'>
+            <ListItem key={id}>
```

Comment:
Why do you need the `ListItem`s here?

Reply:
You right, I don't need it.
Instead, we insert Grid container .